CMakeLists.txt

```diff
@@ -166,15 +166,17 @@ else()
       if (STRIP STREQUAL "STRIP-NOTFOUND")
           message(FATAL_ERROR "strip not found")
       endif()
-
-    else (CMAKE_SYSTEM_NAME STREQUAL Darwin)
-
+    elseif (CMAKE_SYSTEM_NAME STREQUAL Linux)
       # Ensure that objcopy is present
-      find_program(OBJCOPY objcopy)
+      if(DEFINED ENV{CROSSCOMPILE})
+        find_program(OBJCOPY arm-linux-gnueabihf-objcopy)
+      else()
+        find_program(OBJCOPY objcopy)
+      endif()
       if (OBJCOPY STREQUAL "OBJCOPY-NOTFOUND")
           message(FATAL_ERROR "objcopy not found")
       endif()
-    endif (CMAKE_SYSTEM_NAME STREQUAL Darwin)
+    endif ()
```

Comment:
@manu-silicon , :+1:  , I have appended the `if` statement for handling arm64 even though I don't currently follow up the arm64 issue.